library/core/src/ops/function.rs

```diff
@@ -280,6 +280,9 @@ mod impls {
     {
         type Output = F::Output;
 
+        // FIXME: Rust's "try catch" construct (i.e., std::panicking::r#try) use
+        // of FnOnce explicitly violates CFI.
```

Comment:
Is there a way to not have to tag this specific impl with `no_sanitize(cfi)`? Like putting it on something in `panicking.rs` instead? It seems a bit indirect to have to tag this impl for something that lives so far away.

Reply:
Yeah, if we have to tag `FnOnce::call_once` for this, it raises questions as to whether LLVM's CFI implementation can support Rust at all, because this is a **lot** of closures.